src/EditorFeatures/CSharpTest/ImplementInterface/ImplementInterfaceTests.cs

```diff
@@ -7232,9 +7232,9 @@ public async Task TestDoNotReorderComImportMembers_02()
 [Guid(""00000000-0000-0000-0000-000000000000"")]
 interface IComInterface
 {
-    void MOverload() { }
-    void X() { }
-    void MOverload(int i) { }
+    void {|CS0423:MOverload|}() { }
```

Comment:
I'm not sure why this test exists in the IDE at all (or at least with the method name it has) -- the point of the test that was originally cloned in #17927 was that the ordering of the generated class members should (ideally) match the COM import order. But since the test isn't generating any methods at all it's moot -- all this does is assert we don't crash in this case.

Reply:
> But since the test isn't generating any methods at all it's moot 

It does generate property.